splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -2273,16 +2273,22 @@ public static void INVALIDATE_GLOBAL_DICTIONARY_CACHE() throws Exception {
     }
 
     @SuppressFBWarnings(value="SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE", justification="Intentional")
-    public static void SHOW_CREATE_TABLE(String schemaName, String tableName, ResultSet[] resultSet) throws SQLException
+    public static void SHOW_CREATE_TABLE(String schemaName, String tableName, ResultSet[] resultSet) throws SQLException, StandardException
     {
-        List<String> ddls = SHOW_CREATE_TABLE_CORE(schemaName, tableName, false, true);
-        StringBuilder sb = new StringBuilder("SELECT * FROM (VALUES '");
-        sb.append(ddls.get(0));
-        sb.append(";') FOO (DDL)");
-        resultSet[0] = executeStatement(sb);
+        List<String> ddls = SHOW_CREATE_TABLE_CORE(schemaName, tableName, false);
+
+//        There are two solutions to have one place to escape single quotes. During the PR can chosen be the best one:
+//        1) just to replace single quote by double quotes before wrapping to sql
+//        StringBuilder sb = new StringBuilder("SELECT * FROM (VALUES '");
+//        sb.append(ddls.get(0).replace("'", "''"));
+//        sb.append(";') FOO (DDL)");
+//        resultSet[0] = executeStatement(sb);
+
+//        2) to use ProcedureUtils.generateResult
```

Comment:
i guess we can remove this comment, since SELECT * FROM VALUES also has other problems (like fails for long results, see DB-10542)

Reply:
done